features/machine/machine_misc.feature

```diff
@@ -156,6 +156,7 @@ Feature: Machine misc features testing
     When I perform the GET prometheus rest client with:
       | path  | /api/v1/query?                         |
       | query | cloudprovider_vsphere_vcenter_versions |
-    Then the step should succeed
-    And the expression should be true> @result[:parsed]["data"]["result"][0]["metric"]["version"] == "7.0.0"
```

Comment:
You can do it using regexp: `And the expression should be true> @result[:parsed]["data"]["result"][0]["metric"]["version"] =~ /7.0./`

Reply:
Thanks @jhou1 , made the change and validated , its good